packages/crayons-extended/custom-objects/src/components/form-builder/components/fb-field-lookup.tsx

```diff
@@ -201,6 +201,9 @@ export class FbFieldDropdown {
     const boolShowDescription =
       strDescription && strDescription !== '' ? true : false;
 
+    const boolShowRelationshipTypeSelect =
```

Comment:
Why are we checking a hardcoded String here - Can you please move this to form-mapper - config or settings

Reply:
@sangeet-fw  made the changes for the same.